### ISSUE TITLE
The subseq function adds tabs to the sequence headers

### ISSUE THREAD

**Issue by deprekate** [lh3/seqtk#68]
The current version inserts a tab between the header ID and the comment. (I use the @ to show where tabs are)
```
$ seqtk 
Version: 1.0-r82-dirty

$ cat test.fna | tr "\t" "@"
>one comment
ACTG

$ cat test.lst | tr "\t" "@"
two

$ seqtk seq -A test.fna | tr "\t" "@"
>one comment
ACTG

$ seqtk subseq test.fna test.lst | tr "\t" "@"
>one@comment
ACTG
```
Line 569 of seqtk.c uses a tab instead of a space.

**Comment by lh3** [comment]
Like SPACE, TAB is not part of fasta name, either.

**Comment by deprekate** [comment]
Yes, but there the SPACE in a sequence header should not be substituted for a TAB.   It causes problems downstream, especially for software that relies on the SPACE boundary to get the comment portion (which usually includes functional annotation and taxonomy information).

A TAB also messes up other things like, creating TAB delimited lists, and using the header as one column.  It then throws off the column count.

**Comment by tseemann** [comment]
@lh3 I agree with @deprekate that the current behaviour is undesirable.

The vast majority of FASTA/Q files will use a SPACE between the ID and the COMMENT. 

I suspect the TAB wasn't intentional in the FASTA/Q output mode.

```
if (seq->comment.l) printf("\t%s", seq->comment.s);
```
should be
```
if (seq->comment.l) printf(" %s", seq->comment.s);
```

**Comment by lh3** [comment]
Fixed via e5c4fd9. Thanks.
